run_server.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 source activate usi
 
-gunicorn -w 2 -b 0.0.0.0:5000 --timeout 3600 main:app --access-logfile /app/logs/access.log --max-requests 1000
```

Comment:
gunicorn also lets you specify a path via a package

Reply:
I would also suggest renaming the `app.py` to `wsgi.py` but that's a matter of preference